_includes/manuals/2.4/1.2_release_notes.md

```diff
@@ -10,42 +14,28 @@ Additionally, it is stricter in reverse proxy handling. Out of the box Rails acc
-* bondslaves in kickstart kernel options should be string not list ([#31626](https://projects.theforeman.org/issues/31626))
-* Deprecate flot_pie_chart ([#31569](https://projects.theforeman.org/issues/31569))
-* remove ovirt_use_v4 from oVirt compute resource attributes  ([#31551](https://projects.theforeman.org/issues/31551))
-* Improve /puppetrun error message ([#31543](https://projects.theforeman.org/issues/31543))
-* Improve os name matching for icon ([#31531](https://projects.theforeman.org/issues/31531))
-* ALTLinux icons display ([#31491](https://projects.theforeman.org/issues/31491))
-* Provisioning templates - Error message for blank template & fix resetting fields ([#31483](https://projects.theforeman.org/issues/31483))
-* allow ansible inventory template to work without Ansible plugin ([#31482](https://projects.theforeman.org/issues/31482))
-* Allow reinitialization of init values in ForemanForm ([#31461](https://projects.theforeman.org/issues/31461))
-* Clone Operating system ([#31410](https://projects.theforeman.org/issues/31410))
-* Integrate webhooks with ActiveJob ([#31243](https://projects.theforeman.org/issues/31243))
-* Extract the permitted params to a separate variable in registration controller ([#31177](https://projects.theforeman.org/issues/31177))
-* Show host name in ENC output  ([#31168](https://projects.theforeman.org/issues/31168))
-* Use single quotes in registration templates for curl ([#31143](https://projects.theforeman.org/issues/31143))
-* Developer Docs - Create plugin - host.refresh_status & subscriber ([#31139](https://projects.theforeman.org/issues/31139))
-* Add global css registration from plugins ([#30978](https://projects.theforeman.org/issues/30978))
-* Allow Ubuntu 18.04 to netboot with a VLAN tag ([#30737](https://projects.theforeman.org/issues/30737))
-* Add warning banner about puppet ENC extraction ([#30215](https://projects.theforeman.org/issues/30215))
-* Add support for Redfish to BMC Smart Proxy ([#29636](https://projects.theforeman.org/issues/29636))
+####  Personal Access Tokens
+
+WebUI can be used to handle Personal Access Tokens. A new token can be generated by the user and set to expire at a certain time and date.
+To make tokens void before their expiration date and time, one can use the revoke action. ([31080](https://projects.theforeman.org/issues/31080))
+
+* New tab: Hover to current logged in user -> _My Account_ -> _Personal Access Tokens_
+
+#### Support for Redfish to BMC Smart Proxy
 
+The Intelligent Platform Management Interface (IPMI) is being phased out in favour of Redfish for network hardware management. Some of the features for example, boot media aren't available across all vendors (HPE and Dell in particular). The Redfish provider for the BMC Smart Proxy solves these issues while also potentially allowing additional BMC functions to be integrated. ([29636](https://projects.theforeman.org/issues/29636))
```

Comment:
@lzap can you take a look at this one please?

Reply:
Are there plans to phase out IPMI? Not all hardware supports Redfish so I think phasing out is too strong.

Reply:
No there are no plans at the moment, Redfish was just contributed by one of our community users and it is very basic. We currently have not plans on giving up IPMI.

Basically just say that Redfish is now available as another BMC provider.